glue.tf

```diff
@@ -0,0 +1,103 @@
+resource "aws_glue_catalog_database" "default" {
+  name = aws_rds_cluster.default.cluster_identifier
+}
+
+resource "aws_glue_connection" "default" {
+  connection_properties = {
+    JDBC_CONNECTION_URL = "jdbc:${var.engine}://${aws_rds_cluster.default.endpoint}/${aws_glue_catalog_database.default.name}"
```

Comment:
jdbc:postgresql://xxx-cluster.cluster-xxx.us-east-1.rds.amazonaws.com:5432/employee

variablize according to this.